ui/Screen/TransactionDetails.svelte

```diff
@@ -0,0 +1,133 @@
+<script>
+  import { gql } from "apollo-boost";
+  import { getClient, query } from "svelte-apollo";
+  import { pop } from "svelte-spa-router";
+
+  import {
+    identityAvatarUrlStore,
+    identityAvatarFallbackStore,
+    identityDisplayNameStore,
+    identityHandleStore
+  } from "../store/identity.js";
+
+  import { Button } from "../DesignSystem/Primitive";
+  import {
+    ModalLayout,
+    Transaction,
+    TransactionStatusbar
+  } from "../DesignSystem/Component";
+
+  export let params = null;
+
+  const GET_TRANSACTIONS = gql`
+    query Query($ids: [ID!]!) {
+      listTransactions(ids: $ids) {
+        transactions {
+          id
+          state {
+            ... on Applied {
+              block
+            }
+          }
+          timestamp
+          messages {
+            ... on ProjectRegistrationMessage {
+              kind
+              orgId
+              projectName
+            }
+            ... on UserRegistrationMessage {
+              kind
+              id
+              handle
+            }
+          }
+        }
+        thresholds {
+          confirmation
+          settlement
+        }
+      }
+    }
+  `;
+
+  const client = getClient();
+  const transactions = query(client, {
+    query: GET_TRANSACTIONS,
+    variables: {
+      ids: [params.id]
+    },
+    fetchPolicy: "no-cache"
+  });
+
+  const formatMessage = {
+    USER_REGISTRATION: "User registration",
```

Comment:
We should keep types coming from the proxy in one place.

So far we've kept them as constants (have a look how BLOB and TREE are used in components):
https://github.com/radicle-dev/radicle-upstream/blob/master/native/types.js